src/component/app.tsx

```diff
@@ -121,11 +114,50 @@ class App extends React.Component<AppProps> {
 		this.activeTab = id;
 	}
 
+	@MobX.computed
+	protected get isPatternListVisible(): boolean {
+		return Boolean(this.activeTab === App.PATTERN_LIST_ID);
```

Comment:
`Boolean` constructor is not needed here

Reply:
Sure? Maybe necessary for MobX? Not my code BTW :-)

Reply:
Can't imagine a scenarion where `left === right` does not evaluate to `Boolean` ;)